### PR TITLE
Fix test classes

### PR DESCRIPTION
**Pull request by anandjakhaniya**

### CODE REVIEW

File: backend/src/test/java/ai/verta/modeldb/CollaboratorTest.java [hunk 1]
```diff
@@ -142,6 +143,7 @@ public void initializeChannel() throws IOException {
             .build();
   }
 
+  @Ignore
```
Comment (anandjakhaniya) [inline backend/src/test/java/ai/verta/modeldb/CollaboratorTest.java]: Ignore test case due to UAC throws unexpected exception. 
its start to breaking from  https://github.com/VertaAI/verta-uacservice/pull/241/files#diff-e2fb9ba8a921faaaf2762250c3792c850621212db1e77e4b66213619de98d5b5R955 commit. before that it was throwing `NOT_FOUND` for the `NumberFormatException` but now that code change and it is throwing
```
{"thread":"grpc-default-executor-35","level":"DEBUG","loggerName":"ai.verta.uac.db.HibernateFactory","message":"Detected exception of type java.lang.NumberFormatException: For input string: \"github|1234\", which is not categorized as retryable","thrown":{"commonElementCount":0,"localizedMessage":"For input string: \"github|1234\"","message":"For input string: \"github|1234\"","name":"java.lang.NumberFormatException"},"endOfBatch":false,"loggerFqcn":"org.apache.logging.log4j.spi.AbstractLogger","instant":{"epochSecond":1608297108,"nanoOfSecond":564000000},"threadId":89,"threadPriority":5,"source":{"class":"ai.verta.uac.db.HibernateFactory","method":"needToRetry","file":"HibernateFactory.java","line":242}}
```

Comment (anandjakhaniya) [reply backend/src/test/java/ai/verta/modeldb/CollaboratorTest.java]: cc @evgeniilezhnin

Comment (conradoverta) [reply backend/src/test/java/ai/verta/modeldb/CollaboratorTest.java]: Let's fix it in uac instead of ignoring.

**Review by conradoverta — CHANGES_REQUESTED** [review]

File: backend/src/test/java/ai/verta/modeldb/CollaboratorTest.java (hunk 1 above)
Comment (anandjakhaniya) [reply backend/src/test/java/ai/verta/modeldb/CollaboratorTest.java]: Sure, i will remove Ignore. @evgeniilezhnin please take a look on this.

Comment (anandjakhaniya) [reply backend/src/test/java/ai/verta/modeldb/CollaboratorTest.java]: Remove Ingnore from MDB

Comment (evgeniilezhnin) [reply backend/src/test/java/ai/verta/modeldb/CollaboratorTest.java]: https://github.com/VertaAI/verta-uacservice/pull/460

**Review by conradoverta — APPROVED** [review]

**Pull request merged**